apps/api/permissions.py

```diff
@@ -10,6 +17,8 @@
 
 from .models import UserAPIKey
 
+logger = logging.getLogger("api")
```

Comment:
standardise log name:
```suggestion
logger = logging.getLogger("ocs.api")
```